predictoor/pdr-architecture.md

```diff
@@ -0,0 +1,56 @@
+---
+description: >-
+    "We can only see a short distance ahead, but we can see plenty there that needs to be done." — Alan Turing
+---
+
+# Predictoor Architecture
+
+## Privacy and Oasis Sapphire
+Predictoor needs privacy for:
+- Submitted predictions
+- Compute aggregate predictions; and
+- Aggregated predictions — only subscribers can see
+
+This could all be done on fully-centralized infrastructure. But doing so would fail on our other goals: **being globally distributed, censorship resistant, and non-custodial**.
+
+Targeting these needs, we researched & prototyped many privacy technologies. [Oasis Sapphire](https://oasisprotocol.org/sapphire) emerged as the best choice because, as the only privacy-preserving EVM chain in production, it could handle these needs cleanly end-to-end.
+
+## Software Implementation
+Most of [predictoor.ai](https://predictoor.ai) dapp is implemented in the [pdr-web repo](https://github.com/oceanprotocol/pdr-web), with help from [pdr-websocket](https://github.com/oceanprotocol/pdr-websocket/) to fetch feed data and [Ocean Aquarius](https://github.com/oceanprotocol/aquarius) for metadata caching.
+
+The template Predictoor bots, trader bots, trueval bot, and prediction feed publishing are all in the [pdr-backend repo](https://github.com/oceanprotocol/pdr-backend). Contracts are in the Ocean [contracts](https://github.com/oceanprotocol/contracts) repo.
+
+Events emitted by contracts are indexed as Ocean subgraphs, to be consumed by predictoor.ai and the bots. The backend [subgraph README](https://github.com/oceanprotocol/pdr-backend/blob/main/READMEs/subgraph.md) has more info.
+
+## Backend & Contract Details
+The section ["Predictoor Structure”](pdr-intro.md#predictoor-structure) presented much of the Predictoor architecture.  
```

Comment:
Link doesn't work. Change to `(predictoor/pdr-intro#predictoor-structure)`

Reply:
Fixed